From 6f77e19092284114b7fea4ca449b59576b64b744 Mon Sep 17 00:00:00 2001
From: Just_A_Pony <pony@just-a-pony.net>
Date: Fri, 17 Apr 2026 01:16:04 +0800
Subject: [PATCH 10/10] remove greating sticker

---
 .../ui/Components/ChatGreetingsView.java        | 17 +++--------------
 1 file changed, 3 insertions(+), 14 deletions(-)

diff --git a/TMessagesProj/src/main/java/org/telegram/ui/Components/ChatGreetingsView.java b/TMessagesProj/src/main/java/org/telegram/ui/Components/ChatGreetingsView.java
index 207edef29..919417158 100644
--- a/TMessagesProj/src/main/java/org/telegram/ui/Components/ChatGreetingsView.java
+++ b/TMessagesProj/src/main/java/org/telegram/ui/Components/ChatGreetingsView.java
@@ -234,7 +234,7 @@ public class ChatGreetingsView extends LinearLayout {
                 }
             }
         } else {
-            addView(titleView, LayoutHelper.createLinear(LayoutHelper.WRAP_CONTENT, LayoutHelper.WRAP_CONTENT, Gravity.CENTER_HORIZONTAL, 20, 6, 20, 6));
+            addView(titleView, LayoutHelper.createLinear(LayoutHelper.WRAP_CONTENT, LayoutHelper.WRAP_CONTENT, Gravity.CENTER_HORIZONTAL, 20, -2, 20, 6));
             addView(descriptionView, LayoutHelper.createLinear(LayoutHelper.WRAP_CONTENT, LayoutHelper.WRAP_CONTENT, Gravity.CENTER_HORIZONTAL, 20, 6, 20, 6));
             addView(stickerContainer, LayoutHelper.createLinear(112, 112, Gravity.CENTER_HORIZONTAL, 16, 10, 16, 16));
         }
@@ -421,20 +421,9 @@ public class ChatGreetingsView extends LinearLayout {
     @Override
     protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         ignoreLayot = true;
-        if (!preview) {
-            descriptionView.setVisibility(View.VISIBLE);
-        }
-        stickerToSendView.setVisibility(View.VISIBLE);
         super.onMeasure(widthMeasureSpec, heightMeasureSpec);
-        if (getMeasuredHeight() > MeasureSpec.getSize(heightMeasureSpec) && !preview) {
-            descriptionView.setVisibility(View.GONE);
-            stickerToSendView.setVisibility(View.GONE);
-        } else {
-            if (!preview) {
-                descriptionView.setVisibility(View.VISIBLE);
-            }
-            stickerToSendView.setVisibility(View.VISIBLE);
-        }
+        descriptionView.setVisibility(View.GONE);
+        stickerContainer.setVisibility(View.GONE);
         ignoreLayot = false;
         super.onMeasure(widthMeasureSpec, heightMeasureSpec);
     }
-- 
2.53.0

